app/lender/server.py

```diff
@@ -0,0 +1,15 @@
+from flask import jsonify
+
+
+def register_routes(blueprint):
+    @blueprint.route('/get-lender', methods=['GET'])
+    def get_lender():
+
+        result = {
+            "lender_name": "Bank of England Plc",
+            "lender_addr_1": "12 Trinity Place",
```

Comment:
thinking about it we should probably use a standard approach to addresses. [Microformats](http://microformats.org/wiki/adr) defines the properties:
```
post-office-box
extended-address
street-address
locality
region
postal-code
country-name
```
so perhaps:
```
"name": "Bank",
"address": {
  "street-address": "12 Trinity Place",
  "extended-address": "Regents Street",
etc...
}
```

Might be worth doing the same for the other addresses?